recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "2.9.3" %}
+{% set version = "2.9.9" %}
 
 package:
   name: psycopg2
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/p/psycopg2/psycopg2-{{ version }}.tar.gz
-  sha256: 8e841d1bf3434da985cc5ef13e6f75c8981ced601fd70cc6bf33351b91562981
+  sha256: d1454bde93fb1e224166811694d600e746430c006fbb031ea06ecc2ea41bf156
   patches:
     - have_openssl.patch
 
```

Comment:
build number should be reset to 0.
skip py<37 could be added. https://github.com/psycopg/psycopg2/blob/2.9.9/setup.py#L554